package.json

```diff
@@ -59,11 +67,10 @@
     "rowdy": "^0.3.4",
     "sauce-connect-launcher": "^0.13.0",
     "saucelabs": "^1.0.1",
-    "selenium-standalone": "^4.6.3",
+    "selenium-standalone": "4.7.2",
```

Comment:
Current `selenium-standalone` breaks on Node `v0.10.x`, so pin to earlier. See: https://github.com/vvo/selenium-standalone/issues/156